src/mappings/entities/assets/mapAsset.ts

```diff
@@ -559,14 +559,12 @@ export const handleAssetBalanceUpdated = async (event: SubstrateEvent): Promise<
     promises.push(asset.save());
   } else if (updateReason === 'transferred') {
     const details = value as unknown as {
-      readonly instructionId: Option<u64>;
-      readonly instructionMemo: Option<U8aFixed>;
+      instructionId: number | null;
+      instructionMemo: `0x${string}` | null;
     };
 
-    instructionId = details.instructionId?.isSome
-      ? processInstructionId(details.instructionId.unwrap())
-      : null;
-    instructionMemo = bytesToString(details.instructionMemo);
+    instructionId = details.instructionId ? details.instructionId.toString() : '';
```

Comment:
Instead of empty string we should set it to undefined/null. That way querying is easier